src/bucket/test/BucketManagerTests.cpp

```diff
@@ -1242,7 +1243,33 @@ class StopAndRestartBucketMergesTest
             resolveAllMerges(app->getBucketManager().getBucketList());
             auto countersBeforeClose =
                 app->getBucketManager().readMergeCounters();
-            closeLedger(*app);
+
+            if (firstProtocol != secondProtocol &&
```

Comment:
I think we don't need this if block, as it doesn't actually impact upgrades past genesis ledger.

Reply:
oh I see, there's been some renaming, so LEDGER_PROTOCOL_VERSION is the max version supported by core. So yeah, we do need to update it. Why does it need to be here though? I think it'd be cleaner to update it in between application shutdown and start, since this is when the change takes effect.